docs/source/API/containers/ScatterView.rst

```diff
@@ -55,9 +55,11 @@ Description
 
         Constructor from variadic pack of dimension arguments. Constructs ``internal_view`` member.
 
-    .. cppkokkos:function:: ScatterView(::Kokkos::Impl::ViewCtorProp<P...> const& arg_prop, Dims... dims)
+    .. cppkokkos:function:: ScatterView(ALLOC_PROP const& arg_prop, Dims... dims)
 
```

Comment:
TODO:  make `ALLOC_PROC` a Sphinx "type" in this PR & View updates at the same time.  Resolve any conflicts w/ #383.